internal/wire/wirefix/wire_fix.go

```diff
@@ -0,0 +1,118 @@
+package wirefix
```

Comment:
I think that the filename here should be `wirefix.go` to be consistent with the rest of the fixture packages.